Bottom: a57a41b830bdad9601a5a04fec685b96d10d715d
Top:    a4d87fda3c02362c6f1680c2b3b4d9408a488caa
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-02 14:25:46 +0000

Fix distribution of gnet-pcbfwd.scm in the dist tarball

Use the dist_ prefix to make this simple, so our installed
targets are now listed in dist_..._DATA, rather than ..._DATA
and EXTRA_DIST.


---

diff --git a/tools/Makefile.am b/tools/Makefile.am
index 002fecc..d9665a9 100644
--- a/tools/Makefile.am
+++ b/tools/Makefile.am
@@ -5,12 +5,10 @@
 ## in ${exec_prefix}/bin  (by bin_SCRIPTS and bin_PROGRAMS)
 
 toolsdir=	$(pkgdatadir)/tools
-tools_DATA=	${TOOLS}
+dist_tools_DATA=	${TOOLS}
 
 scmdatadir = $(datadir)/gEDA/scheme
-scmdata_DATA = $(DIST_SCM)
-
-EXTRA_DIST= ${TOOLS}
+dist_scmdata_DATA = $(DIST_SCM)
 
 bin_SCRIPTS= \
 	MergePCBPS \
